.github/autosubmit.yml

```diff
@@ -0,0 +1,16 @@
+default_branch: 'main'
```

Comment:
In yaml, main is fine. None of these need to be string encapsulated as it's implicit